transform/models/marts/geo_reference/_geo_reference__models.yml

```diff
@@ -39,6 +39,9 @@ models:
           and the tabulation block number
       - name: geometry
         description: The spatial component of geographic features
+    tests:
+      - count_equals_other:
+          other: source('building_footprints', 'california_building_footprints')
```

Comment:
Great to see this logic here! I haven't dove into tests yet so this can be a helpful reference.

Reply:
I cribbed this test from one that @AeriShan-ODI wrote in the CalHR repo :)